guide/start/policies.md

```diff
@@ -435,14 +435,14 @@ you could use a load generator like jmeter, or use a script such as the one show
 (changing URL for the URL of your load-balancer):
 
 {% highlight bash %}
-$ URL=http://10.10.10.101:8000/
+$ URL=http://10.10.10.101:8080/
```

Comment:
I thought nginx would come up on 8000, rather than 8080 (which is the port that the first tomcat would use).